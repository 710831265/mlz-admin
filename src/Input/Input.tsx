import React, { useState, useContext } from 'react';
import { Input as AntdInput } from 'antd';

const Input = AntdInput;

export default Input;
