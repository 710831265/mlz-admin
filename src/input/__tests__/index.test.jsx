import React, { Component } from 'react';
import testMount from '../../../tests/testMount';
import Input from '..';

describe('ğŸ“¦  Input', () => {
  testMount(Input);
});
