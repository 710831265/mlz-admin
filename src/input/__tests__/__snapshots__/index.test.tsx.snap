// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª Input snapshot test snapshot comparing 1`] = `"<input type=\\"text\\" class=\\"ant-input\\" value>"`;
